assets/queries/cloudFormation/viewer_protocol_only_https_compliant/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "viewer_protocol_only_https_compliant",
+  "queryName": "Viewer Protocol Only HTTPS Compliant",
```

Comment:
The name should reflect the vulnerability. A more suitable name would be: "Viewer Protocol Policy Allows HTTP Protocol"